spec/helpers/multi_selected.data.js

```diff
@@ -22,6 +22,7 @@ var multi_test_data = [
 				name: "Open Source",
 				sub: [
 					{
+            id: 41,
```

Comment:
If you have indentation level issue here - it's my IDE or GHub issue. What I have is:

![idents](https://cloud.githubusercontent.com/assets/8696264/11339197/0f85f4c4-9201-11e5-830e-8df725f12d67.jpg)

And all is as should be.